commit 1d45a1987073ad1e96a490993cefdfec7088c1c3 (refs/changes/36/5236/2)
Author: Rui Loura <rui@joyent.com>
Date:   2018-12-13T19:19:26+00:00 (10 months ago)
    
    TRITON-1033 adminIpFromSysinfo should check for missing NIC
    Reviewed by: Robert Bogart <Robert.bogart@joyent.com>
    Approved by: Robert Bogart <Robert.bogart@joyent.com>

diff --git a/lib/index.js b/lib/index.js
index f9a8686..f0ae562 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -83,7 +83,11 @@ function adminIpFromSysinfo(sysinfo) {
 
     var nic = adminNicFromSysinfo(sysinfo);
 
-    return nic.ip4addr;
+    if (nic) {
+        return nic.ip4addr;
+    }
+
+    return null;
 }
 
 // ---- From VM Metadata
